### ISSUE TITLE
Barcode starting with 0 not detected when EAN13 and UPC A are both enabled

### ISSUE THREAD

**Issue by jneuhaus20** [zxing-js/library#457]
I have a barcode starting with '0' that is either EAN13 or UPC A (honestly not sure which one it "really" is, just that it's ambiguous according to the readers.)  When EAN13 and UPC_A are both enabled, no barcode is detected.

I tracked it down to the conversion functionality here:
https://github.com/zxing-js/library/blob/b88907ccc4cfb3f8dec80650afdc8c8f99d771f9/src/core/oned/MultiFormatUPCEANReader.ts#L97-L109

The problem is that `UPCEANReader` doesn't actually set the raw bytes, so `rawBytes` is set to `null` at line 98, and the accessor on 103 throws an error, which is caught higher up and converted into `NotFoundException`.

I tried to fork the demo's StackBlitz for this as the current one is out of date, but there's an unrelated ES5/ES6 issue going on across the packages.

I created PR #456 for this already.

**Comment by ahmedhasanmotiwala** [comment]
I scan UPC-A Barcode i.e 725272730706 and its detecting EAN_13 with a leading 0 in this barcode i.e 0725272730706

**Comment by jneuhaus20** [comment]
Yeah, the graphical encoding (bars) are literally that (see [this great explanation on the two](https://www.barcodestalk.com/resource/what-difference-between-upc-and-ean)).  Basically, US and Canada just don't write the leading 0 under the bars, for historical reasons that are stupid.
